.github/renovate.json

```diff
@@ -20,6 +14,19 @@
         "enabled": true
       },
       "osvVulnerabilityAlerts": true
+    },
+    {
+      "matchBaseBranches": ["/^release-[0-9]+\\.[0-9]+$/"],
+      "matchManagers": ["dockerfile"],
+      "matchPackageNames": ["/registry\\.access\\.redhat\\.com\\/ubi[0-9]+\\/ubi-minimal/"],
+      "managerFilePatterns": ["build/Dockerfile.rhtap"],
+      "ignorePaths": ["*"],
+      "enabled": true,
+      "pinDigests": true
+    },
+    {
+      "matchManagers": ["dockerfile"],
+      "enabled": false
```

Comment:
Should the order of these be reversed? From https://docs.renovatebot.com/configuration-options/#packagerules
> [!TIP]
> Order your packageRules so the least important rules are at the top, and the most important rules at the bottom. This way important rules override settings from earlier rules if needed.